The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

The database cluster will be initialized with locale "en_US.utf8".
The default database encoding has accordingly been set to "UTF8".
The default text search configuration will be set to "english".

Data page checksums are disabled.

fixing permissions on existing directory /var/lib/postgresql/data/pgdata ... ok
creating subdirectories ... ok
selecting default max_connections ... 100
selecting default shared_buffers ... 128MB
selecting dynamic shared memory implementation ... posix
creating configuration files ... ok
running bootstrap script ... ok
performing post-bootstrap initialization ... ok
syncing data to disk ... ok

Success. You can now start the database server using:

    pg_ctl -D /var/lib/postgresql/data/pgdata -l logfile start


WARNING: enabling "trust" authentication for local connections
You can change this by editing pg_hba.conf or using the option -A, or
--auth-local and --auth-host, the next time you run initdb.
waiting for server to start....2025-02-14 08:49:22.532 UTC [43] LOG:  listening on IPv6 address "::1", port 5432
2025-02-14 08:49:22.532 UTC [43] LOG:  listening on IPv4 address "127.0.0.1", port 5432
2025-02-14 08:49:22.534 UTC [43] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 08:49:22.542 UTC [44] LOG:  database system was shut down at 2025-02-14 08:49:22 UTC
2025-02-14 08:49:22.546 UTC [43] LOG:  database system is ready to accept connections
 done
server started
CREATE DATABASE

CREATE ROLE


/usr/local/bin/docker-entrypoint.sh: ignoring /docker-entrypoint-initdb.d/*

waiting for server to shut down....2025-02-14 08:49:22.930 UTC [43] LOG:  received fast shutdown request
2025-02-14 08:49:22.931 UTC [43] LOG:  aborting any active transactions
2025-02-14 08:49:22.933 UTC [43] LOG:  worker process: logical replication launcher (PID 50) exited with exit code 1
2025-02-14 08:49:22.933 UTC [45] LOG:  shutting down
2025-02-14 08:49:22.943 UTC [43] LOG:  database system is shut down
 done
server stopped

PostgreSQL init process complete; ready for start up.

2025-02-14 08:49:23.039 UTC [1] LOG:  listening on IPv4 address "0.0.0.0", port 5432
2025-02-14 08:49:23.039 UTC [1] LOG:  listening on IPv6 address "::", port 5432
2025-02-14 08:49:23.040 UTC [1] LOG:  listening on Unix socket "/var/run/postgresql/.s.PGSQL.5432"
2025-02-14 08:49:23.047 UTC [76] LOG:  database system was shut down at 2025-02-14 08:49:22 UTC
2025-02-14 08:49:23.050 UTC [1] LOG:  database system is ready to accept connections
2025-02-14 08:49:31.216 UTC [90] ERROR:  relation "_prisma_migrations" does not exist at character 126
2025-02-14 08:49:31.216 UTC [90] STATEMENT:  SELECT "id", "checksum", "finished_at", "migration_name", "logs", "rolled_back_at", "started_at", "applied_steps_count" FROM "_prisma_migrations" ORDER BY "started_at" ASC
2025-02-14 08:49:31.570 UTC [90] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 08:49:31.570 UTC [90] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_81bda8e8-abeb-410f-83b4-00dc7af71087" WITH (FORCE)
2025-02-14 08:49:32.047 UTC [90] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 08:49:32.047 UTC [90] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_3eb0af16-96af-4cd7-bf5e-5903bdcc4200" WITH (FORCE)
2025-02-14 08:49:32.674 UTC [90] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 08:49:32.674 UTC [90] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_9364e42f-e39f-4c33-99af-998155edec87" WITH (FORCE)
2025-02-14 08:49:33.142 UTC [90] ERROR:  syntax error at or near "WITH" at character 89
2025-02-14 08:49:33.142 UTC [90] STATEMENT:  DROP DATABASE IF EXISTS "prisma_migrate_shadow_db_09b4ad68-6ca4-486d-8456-c0ff7db065a1" WITH (FORCE)
